### DISCUSSION TITLE
For better upscaling with Fooocus MRE

### DISCUSSION THREAD

**Discussion by kubilaykilinc** [MoonRide303/Fooocus-MRE#116] [General]
I am using the Fooocus MRE version. When I upscale images using Fooocus, some changes occur in the images. To solve this, I follow these steps:

1. I create the image. I keep all the settings, models, and Loras used to create the prompt or image.
2. I use this image as a source by sending it to input with output to input in the image2image tab.
3. I activate both control lora canny and control lora depth in the CN tab. (I use the default settings.)
4. I activate the Enhance Image checkbox and drag and drop the image I created in step 1 here. I select upscale 2x and click to generate. (uncheck RANDOM seed if it checked. Use same seed number) 

I can repeat the same process again to upgrade to 4K.
However, even though my computer has 32 GB of RAM and 12 GB of NVIDA, Fooocus freezes after completing this process. (It makes the image 4K and then freezes.) Maybe this issue will be resolved soon.

If you want to upscale a work you have done before, you can follow the same steps. You just need to load the image with Load Prompt in the first step. After checking the prompt and your settings, you need to load the image into image 2 image with Load image(s) input. The other steps will be the same as I wrote above.

I did a perfect upscale in this way. There was no distortion. And the upgrade was done with perfect details. When I don't apply this process, things like eyes shifting (cross-eyed, etc.) occur.

**Comment by barepixels** [comment]
I seem to recall upscale 2x FAST doesn't add distortion.  No need to activate activate both control lora canny and control lora depth in the CN tab.  No need to use input in the image2image tab.  I don't think you even need prompts

Can you confirm this?

**Reply by 3Diva** [reply]
@barepixels Yeah, if you're just wanting to upscale with zero changes then 2X Fast seams to do just that. Honestly there are barely any changes to the image even with the regular 2X upscaler, so I'm not sure why one would need to go through image2image and canny and all that.
